### ISSUE TITLE
Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[text()='Remove tag']"}

### ISSUE THREAD

**Issue by plgonzalezrx8** [weskerfoot/DeleteFB#146]
Keep getting that error for both Remove tag and Hide from timeline. Any idea on how can I fix this? Thanks.

**Comment by weskerfoot** [comment]
It will be something FB updated most likely, unfortunately. I will see if I can debug it.

**Comment by plgonzalezrx8** [comment]
Yes sir, I am on the new front end, they disabled any way to go back to classic, at least on my end.

**Comment by gitrndm** [comment]
Running into the same problem, can't switch back to classic FB because it went offline in September. Any fix in sight? Thank you in advance!

**Comment by weskerfoot** [comment]
> Running into the same problem, can't switch back to classic FB because it went offline in September. Any fix in sight? Thank you in advance!

I'll try to find some time to look at it tomorrow

**Comment by weskerfoot** [comment]
This will be fixed in the next release
